runtime/thread/threadshared.mk.ftl

```diff
@@ -95,6 +95,7 @@ endif # Linux
 
 ifeq (osx,$(OMR_HOST_OS))
 GLOBAL_LDFLAGS += -install_name lib$(MODULE_NAME).dylib
+GLOBAL_LDFLAGS += -compatibility_version 1.0.0 -current_version 1.0.0
 endif # OSX
```

Comment:
All of this should be removed: the `-install_name` option is superseded by OMR (e.g. [here](https://github.com/eclipse/omr/blob/master/omrmakefiles/rules.osx.mk#L125)) and the version options are handled by eclipse/omr#5251.

Reply:
Removed it.